### PR TITLE
Grafana to get prometheus metrics

### PR DESCRIPTION
**Pull request by akshtshrma24**
The initial dashboard for Quasar container monitoring:
image

### CODE REVIEW

**Review by neelp03 — APPROVED** [review]
I am gonna trust you on this one

**Comment by starphys** [comment]
This has a nearly duplicate provisioning folder at the top level, we should just keep the one in grafana and get rid of the duplicate.

File: ledsign/LEDSignHandler.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ setInterval(async () => {
   if (!data) {
     return;
   }
-
+
```
Comment (evanugarte) [inline ledsign/LEDSignHandler.js]:
```suggestion

```

File: monitoring/grafana/provisioning/dashboards/all.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+apiVersion: 1
+
+providers:
+  - name: "Grafana"
+    org_id: 1
+    folder: "Services"
+    type: "file"
+    options:
+      path: "/etc/grafana/provisioning/dashboards"
```
Comment (evanugarte) [inline monitoring/grafana/provisioning/dashboards/all.yml]:
```suggestion
      path: "/etc/grafana/provisioning/dashboards"

```

**Review by evanugarte — APPROVED** [review]
Tested locally and this works with `docker-compose up`, visiting http://localhost:3000

File: ledsign/LEDSignHandler.js (hunk 1 above)
Comment (evanugarte) [inline ledsign/LEDSignHandler.js]:
```suggestion

```

**Pull request merged**